blockchain/v2/reactor_test.go

```diff
@@ -494,7 +494,9 @@ func newReactorStore(
 				lastBlockMeta.BlockID,
 				state.Validators,
 				privVals[0],
-				lastBlock.Header.ChainID)
+				lastBlock.Header.ChainID,
+				time.Now(),
```

Comment:
This needs to import `"time"` in the file.

Reply:
Thanks!